src/script.rs

```diff
@@ -1158,8 +1158,7 @@ mod test {
         assert!(script_2 < script_3);
         assert!(script_3 < script_4);
 
-        assert!(script_1 <= script_1);
```

Comment:
These lines should not be deleted

Reply:
Just curious, what's this supposed to catch? Doesn't the compiler enforce that `script_1` is equal to `script_1`? I've replaced it with `assert_eq`, but maybe you actually wanted the original assertions. Let me know.

Reply:
I think the test is script is for `script_ord` which is supposed to check basic comparison orderings like `script_1 >= script_1`. Even if it tests something that is trivial, it should not be removed as a part of this PR.

Reply:
I've reverted it back to the original state.